linode/manager
# Update to`vite@4.3.9` for `CVE-2023-34092`

## Description 📝
- Update to `vite@4.3.9` for CVE-2023-34092
- Also updates Vite related dependencies

## How to test 🧪
- Run `yarn` to ensure you have updated dependencies
- Verify that you can start the local dev server with `yarn dev`
- Verify that CI / e2es pass
